doc/bluealsa-plugins.7.rst

```diff
@@ -416,13 +419,28 @@ CTL Parameters
     selects the most recently connected device.
 
   EXT
-    Causes the plugin to include controls for codec and software volume
-    selection. If the value is **yes** then these additional controls are
-    included. The default is **no**. The soft volume controls are called "Mode"
-    and take values "software" and "pass-through"; the playback control has
-    index 0 and capture control index 1. See ``bluealsa(8)`` for more on the
-    soft volume setting , and `Codec selection`_ in the **NOTES** section below
-    for more information on the Codec control.
+    Causes the plugin to include controls for codec, delay adjustment and/or
+    software volume selection. If the value is **yes** then all of these
+    additional controls are included; if the value is **no** then none of them
+    are included. The default is **no**. This parameter can also select a
+    single control, or a colon (':') separated list of controls by using the
+    values **codec**, **sync**, or **softvol**. For example:
```

Comment:
I like the idea of colon separated list! Maybe BAT option should be merged with the EXT? I will experiment with that. If it "feels" OK I'd like to cherrypick this change before merging delay adjustment.

Reply:
I've just cherry-picked this feature into the https://github.com/arkq/bluez-alsa/tree/ext branch. I'd be glad if you could review it if I didn't mess up anything :D I know that it's somehow backward incompatible, but I think that we can live with that.

Reply:
I'll try to do a review in this coming week.